recipes/daml/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+python:
```

Comment:
You should not be specifying the Python via this technique. We have a global pinning for that.